deploy/crds/atlassian.com_nodegroups_crd.yaml

```diff
@@ -306,6 +306,12 @@ spec:
                   - triggerEndpoint
                   type: object
                 type: array
+              priority:
+                description: Priority controls the ordering of CNR creation for this
+                  NodeGroup. Lower values are higher priority. 0 runs first, then
```

Comment:
Similar to @mwhittington21 suggestion for the comment, this should just say that any number is usable